### PR TITLE
Fix readme issues and duplicated licence lines issue

### PR DESCRIPTION
**Pull request by kazrael2119**
fixes https://github.com/Azure/autorest.typescript/issues/2882
fixes https://github.com/Azure/autorest.typescript/issues/2865
fixes https://github.com/Azure/autorest.typescript/issues/2864

### CODE REVIEW

**Review by qiaozha — APPROVED** [review]

File: packages/typespec-ts/static/static-helpers/pagingHelpers.ts [hunk 1]
```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft Corporation.
```
Comment (MaryGao) [inline packages/typespec-ts/static/static-helpers/pagingHelpers.ts:1]: These codes are till in our emitter repo, as files I think we still need these headers.

Comment (kazrael2119) [reply packages/typespec-ts/static/static-helpers/pagingHelpers.ts:1]: emitter will add these two line from https://github.com/kazrael2119/autorest.typescript/blob/b61ee35905401ede30f1597ce49b410dd4a4d732/packages/typespec-ts/src/utils/emitUtil.ts#L61-L65

Comment (MaryGao) [reply packages/typespec-ts/static/static-helpers/pagingHelpers.ts:1]: @lirenhe do you know the guidance here? do we need to add license headers for these static resources? these codes are files not under `src` folder but `static` [folder ](https://github.com/Azure/autorest.typescript/tree/main/packages/typespec-ts/static). And we would copy these files into publish artifacts as static resources also.

FYI - the generated code would always have lisence header.

Comment (MaryGao) [reply packages/typespec-ts/static/static-helpers/pagingHelpers.ts:1]: image

**Pull request merged**